config/crd/experimental/gateway.networking.k8s.io_backendtlspolicies.yaml

```diff
@@ -155,22 +187,33 @@ spec:
                     maxItems: 8
                     type: array
                   hostname:
-                    description: "Hostname is used for two purposes in the connection
-                      between Gateways and backends: \n 1. Hostname MUST be used as
-                      the SNI to connect to the backend (RFC 6066). 2. Hostname MUST
-                      be used for authentication and MUST match the certificate served
-                      by the matching backend. \n Support: Core"
+                    description: |-
+                      Hostname is used for two purposes in the connection between Gateways and
+                      backends:
+
+
+                      1. Hostname MUST be used as the SNI to connect to the backend (RFC 6066).
+                      2. Hostname MUST be used for authentication and MUST match the certificate
+                         served by the matching backend.
+
+
+                      Support: Core
```

Comment:
This has a surprisingly large diff for CRDs, but as far as I can tell, all our no-ops like this. No action needed here, just leaving a comment for posterity to explain why we merged such a large change for a dep upgrade.

Reply:
Thank you for the suggestion.